src/app/app.component.ts

```diff
@@ -15,31 +16,35 @@ export class ClickerApp {
   private platform: Platform;
   private splash: SplashScreen;
   private status: StatusBar;
+  private translateService: TranslateService;
 
   public rootPage: any;
   public pages: Array<{ title: string, component: any }>;
 
-  constructor(platform: Platform, menu: MenuController, splash: SplashScreen, status: StatusBar) {
+  constructor(platform: Platform, menu: MenuController, splash: SplashScreen, status: StatusBar, translateService: TranslateService) {
 
     this.menu = menu;
     this.platform = platform;
     this.splash = splash;
     this.status = status;
+    this.translateService = translateService;
 
-    this.rootPage = ClickerList;
     this.initializeApp();
-
-    // set our app's pages
-    this.pages = [
-      { title: 'Clickers', component: ClickerList },
-      { title: 'Goodbye Ionic', component: Page2 },
-    ];
   }
 
-  private initializeApp(): void {
-    this.platform.ready().then(() => {
+  private initializeApp(): Promise<any> {
+    return this.platform.ready().then(() => {
+      this.translateService.setDefaultLang('en');
+      this.translateService.use('en');
+
       // Okay, so the platform is ready and our plugins are available.
       // Here you can do any higher level native things you might need.
+      this.rootPage = ClickerList;
+      // set our app's pages
+      this.pages = [
+        {title: 'MENU.CLICKERS', component: ClickerList},
+        {title: 'MENU.PAGE2', component: Page2},
+      ];
```

Comment:
Presumably the translateService needs this.platform.ready() for some reason.

Does all the other code need moving in here?

I try to keep this all as close as possible to Ionic's tutorial starter as it makes it easier when doing the upgrades.

If it all needs to be here that's fine, but the comment probably needs to be at the top?